.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "depbot"
+  - package-ecosystem: "pip"
+      directory: "/ztp/resource-generator/tools/cachito"
+      schedule:
+        interval: "weekly"
```

Comment:
why not daily?

Reply:
Thought there might be too many PRs but a daily check on keeping dependencies up to date seems like a good idea none the less.. updated..